pkg/operator/ceph/csi/template/csi-logrotate-sidecar.yaml

```diff
@@ -28,6 +28,10 @@ command:
 image: {{ .CSIPluginImage }}
 imagePullPolicy: IfNotPresent
 name: log-collector
+{{ if and .Privileged .CSILogRotation }}
```

Comment:
any specific reason to have `and` check here, can we use `.Privileged` only

Reply:
yaa e already check for logrotate before appending the container so no need, updated the code

Reply:
@parth-gr ^^

Reply:
it would be good to add that for kubernetes (vanilla ) as well like we added above

Reply:
Done